targets/strainGauge/main.cpp

```diff
@@ -0,0 +1,55 @@
+/**
+* This example prints out stress reading from a strain gauge
+*/
+
+#include <EVT/io/ADC.hpp>
+#include <EVT/io/UART.hpp>
+#include <EVT/manager.hpp>
+#include <EVT/utils/time.hpp>
+#include <SGM.hpp>
+
+namespace IO = EVT::core::IO;
+namespace DEV = EVT::core::DEV;
+namespace time = EVT::core::time;
+
+/**
+* Note, this conversion function is just to show off how a conversion
+* function will look like. In actuality, this will be determined through
+* testing/reading the strain gauge datasheet.
+*/
+uint32_t convert(uint32_t voltage) {
+    return voltage / 2;
+}
+
+int main() {
+    EVT::core::platform::init();
+
+    // Setup IO
+    IO::UART& uart = IO::getUART<IO::Pin::PA_6, IO::Pin::PA_5>(9600);
+
+    // Setup pins for Strain Gauges
+    IO::ADC& adc0 = IO::getADC<IO::Pin::PC_0>();// Strain Gauge D
+    IO::ADC& adc1 = IO::getADC<IO::Pin::PC_1>();// Strain Gauge C
+    IO::ADC& adc2 = IO::getADC<IO::Pin::PC_2>();// Strain Gauge B
+    IO::ADC& adc3 = IO::getADC<IO::Pin::PC_3>();// Strain Gauge A
+
+    // Create an instance for each strain gauge
+    SGM::DEV::StrainGauge gauges[NUM_GAUGES] = {
+        SGM::DEV::StrainGauge(adc0, convert),
+        SGM::DEV::StrainGauge(adc1, convert),
+        SGM::DEV::StrainGauge(adc2, convert),
+        SGM::DEV::StrainGauge(adc3, convert),
+    };
+
+    SGM::SGM sgm = SGM::SGM(gauges);
```

Comment:
This example is just for  the strain gauge driver, so you don't need to instantiate the `SGM`
```suggestion
```